build.gradle

```diff
@@ -298,7 +298,7 @@ task nightlyResources(type: ProcessResources) {
 task nightlyRelease(type: ShadowJar) {
 	from sourceSets.main.output
 	dependsOn nightlyResources, licenseMain
-	archiveName = 'Skript-nightly.jar'
+	archiveName = 'Skript.jar'
```

Comment:
You can keep the old name, and remove the rename in the workflow

Reply:
The Skript runner requires Skript.jar otherwise it has no clue what the name of the jar will be. I tried to rename each reference, but it still didn't work. Skript.jar is perfectly fine.